src/chrome/content/rules/Coletivos.org.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="Coletivos.org">
+	<target host="coletivos.org" />
```

Comment:
Could you also add `www.coletivos.org` as a target with a special redirect rule `<rule from="^http://www\.coletivos\.org/" to="https://coletivos.org/"> and a comment explaining why this is necessary (invalid certificate) please?

Reply:
Done